client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -71,6 +73,11 @@ body {
   }
 }
 
+.form-group.required .control-label:after {
```

Comment:
Why override here instead of update `form.less`?

Reply:
Fixing it!